.github/workflows/ci.yml

```diff
@@ -183,16 +196,36 @@ jobs:
           target: ${{ matrix.target }}
           profile: minimal
 
+      - name: caching
+        uses: Swatinem/rust-cache@v1
+        with:
+          key: cts-a # suffix for cache busting
+          working-directory: wgpu/cts_runner
+          target-dir: wgpu/cts_runner/target
+
+      # We enable line numbers for panics, but that's it
```

Comment:
is this to reduce the cache size, or for something else?

Reply:
Yeah it is, plus reducing IO load a bit, hopefully making things a bit faster.